app.js

```diff
@@ -0,0 +1,56 @@
+const {
+  execTerraform,
+  parseVars,
+  makeVarFile,
+} = require("./helpers");
+
+async function executeTerraformInit(action, settings) {
+  return exeTf("init", action, settings);
+}
+
+async function executeTerraformApply(action, settings) {
+  return exeTf("apply", action, settings);
+}
+
+async function executeTerraformPlan(action, settings) {
+  return exeTf("plan", action, settings);
+}
+
+async function executeTerraformDestroy(action, settings) {
+  return exeTf("destroy", action, settings);
+}
+
+async function exeTf(mode, action, settings) {
+  const workDir = action.params.workingDir || settings.workingDir;
```

Comment:
My habit is to progressively shorter var names for progressively short-lived variables, and for vars that live only in a small block, unintelligibly short names like wkd are even acceptable. But that might be a bad habit. I do run into regular issues with the linter maximum of 100? characters per line, but there are other ways to avoid that I'm aware. I shall adjust my habits. DONE

Reply:
done in both PRs